cluster-up/cluster/k8s-1.17/README.md

```diff
@@ -23,15 +23,6 @@ node01    NotReady   master    31s       v1.17.1
 node02    NotReady   <none>    5s        v1.17.1
 ```
 
-## Bringing the cluster up with cluster-network-addons-operator provisioned
```

Comment:
Why do you drop the documentation? I find it useful.

Reply:
I have move it.